templates/main.tftpl

```diff
@@ -1 +1,34 @@
-#This file is where the code for your application will be in the future.
\ No newline at end of file
+resource "azurerm_resource_group" "network" {
+  name     = "rg-$${local.application_name}Network-$${local.stage}-$${local.env_num}"
```

Comment:
```suggestion
  name     = "rg-$${local.application_name}Network-$${local.stage}-01"
```
Why use the env_num here? I think resource groups should just always start at 01 since they can have the same name across subscriptions anyway.

Reply:
yes,, we prefer to have all values as we can managed by variables and follow a standar of code.

Reply:
What I meant was that our naming convention currently just says to always reset this counter back to 01 for each subscription. So for example if we have the subscription `prd-MyApplication-02` the resource group should still start at `rg-MyApplicationNetwork-prd-01`.

If we want to change this and make the RG names unique across each tenant we can. But in that case we should change the naming convention too, which in turn would mean all previous deployments are now technically incorrect. We could still do that but in my opinion we should only make such a large change if it's 100% necessary.